sources/Delimiter.swift

```diff
@@ -59,3 +59,6 @@ extension Delimiter {
         }
     }
 }
+
+let newline = "\n".unicodeScalars
+let newlineArray = newline.map{ $0 }
```

Comment:
I don't think it is a good idea to have global variables. Moreover when they don't have a `private` or `fileprivate` modifier.